hpack.cabal

```diff
@@ -31,6 +31,7 @@ executable hpack
     , deepseq
     , directory
     , filepath
+    , split
```

Comment:
I think `text` also provides the functionality we need from `split` and is already a transitive dependency, so I would prefer to use that.